solrmonitor/solrmonitor.go

```diff
@@ -412,18 +412,6 @@ func (c *SolrMonitor) getCollNameFromPath(path string) string {
 func (c *SolrMonitor) start() error {
 	// Synchronously check the initial calls, then setup event listening.
 
-	// Ensure no state format v1 collections exist.
-	globalClusterStatePath := c.solrRoot + "/clusterstate.json"
-	globalClusterState, _, err := c.zkCli.Get(globalClusterStatePath)
-	if err != nil {
```

Comment:
Should we just change this such that if the clusterstate.json is missing then we continue? My thought is that given gosolr is open source it might be best to maintain backward compatibility and check for either len > 2 or clusterstate.json is missing.

Reply:
I just started solr 8.8 locally, and it seems it `clusterstate.json` file in not in zk then it creates. I can verify this one more time

Reply:
looks like solr 8.8 just recreate `clusterstate.json` file if its not there in zookeeper.
```
(CONNECTED [localhost:2181]) /solr> rmr clusterstate.json
(CONNECTED [localhost:2181]) /solr> ls
aliases.json
autoscaling
autoscaling.json
clusterprops.json
collections
configs
fs_config_updates
live_nodes
live_query_nodes
node_roles
overseer
overseer_elect
roles.json
security.json
(CONNECTED [localhost:2181]) /solr> ls
aliases.json
autoscaling
autoscaling.json
clusterprops.json
clusterstate.json
collections
configs
fs_config_updates
live_nodes
live_query_nodes
node_roles
overseer
overseer_elect
roles.json
security.json
(CONNECTED [localhost:2181]) /solr> json_cat clusterstate.json
{}

```